src/monitor/monitor-client.c

```diff
@@ -157,8 +165,10 @@ static int ProcessMonitor(jack_nframes_t nframes, void *arg)
             {
                 for (jack_nframes_t i=0; i<nframes; ++i)
                 {
-                    bufOut1[i] = bufIn1[i] * volume;
-                    bufOut2[i] = bufIn2[i] * volume;
+                    if (floats_differ_enough(volume, smooth_volume))
```

Comment:
this is not too good, comparing the volume every single frame sounds expensive.
I think it is best to remove this comparisons, leaving only the one at the end of the audio cycle.